notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/Installation.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+/**
+ * Wraps a record of a device, for use resolving future requests to broadcast notifications.
+ */
+public class Installation {
+    private String pushChannel;

```

Comment:
Let's be consistent with our member variables, prepending `m` for example `mPushChannel`

Reply:
Fixed in 583cdf30d92d7f9a3962e8b37fae21d9037b4511